enterprise/backend/src/metabase_enterprise/sandbox/models/group_table_access_policy.clj

```diff
@@ -77,6 +77,24 @@
                          :expected    table-col-base-type
                          :actual      (:base_type col)}))))))
 
+(defenterprise add-sandboxes-to-permissions-graph
+  "Augment a provided permissions graph with active sandboxing policies."
+  :feature :sandboxes
+  [graph]
+  (m/deep-merge
+   graph
+   (let [sandboxes (t2/select :model/GroupTableAccessPolicy
+                              {:select [:s.group_id :s.table_id :t.db_id :t.schema]
+                               :from [[:sandboxes :s]]
+                               :join [[:metabase_table :t] [:= :s.table_id :t.id]]})]
+     (reduce (fn [acc {:keys [group_id table_id db_id schema]}]
+               (assoc-in acc
+                         [group_id db_id :data :schemas schema table_id]
+                         {:query :segmented
+                          :read :all}))
+             {}
+             sandboxes))))
+
```

Comment:
Glad to see this, and it looks very straightforward, but does it belong in this PR? It's not really mentioned.

Reply:
This was already in the feature branch